lambda/src/main/scala/pricemigrationengine/services/CohortTableTest.scala

```diff
@@ -25,6 +26,8 @@ object CohortTableTest {
           for {
             _ <- console.putStrLn(s"Updating cohort table with result: $result")
           } yield ()
-    }
+
+        def update(result: SalesforcePriceRiseCreationResult): ZIO[Any, CohortUpdateFailure, Unit] = ???
+      }
   )
 }
```

Comment:
We can probably get rid of the `CohortTableTest` or move it to the test codebase